src/app/workflow/workflow-detail/workflow-detail.component.ts

```diff
@@ -391,6 +349,114 @@ export class WorkflowDetailComponent implements OnInit, OnDestroy {
       });
   }
 
+
+  openCopy(content, jobId: string) {
+    this.getJob(content, jobId);
+  }
+
+  getJob(content, jobId: string ) {
```

Comment:
The name could be a bit more explicit about what the function is doing, something like `populateAndOpenCopyModal` for example